accepted/2020/linking-libraries.md

```diff
@@ -0,0 +1,152 @@
+# Linking the .NET Libraries
+
+When publishing a self-contained application, the .NET Core runtime is bundled together with the application. This bundling adds a significant amount of content to the packaged application. Depending on the complexity of the application, only a subset of the runtime is required to run the application. These unused parts of the runtime are unnecessary and can be trimmed from the packaged application.
+
+This is especially important for applications that need to be downloaded over the internet, for example Xamarin and Blazor applications. End users on a slow or limited network will be less likely to install and use applications that are too large.
+
+The [ILLinker](https://github.com/mono/linker) is a tool that is able to take an application and trim the classes and methods that are not used by the application. In many scenarios, this results in a significantly smaller packaged application.
```

Comment:
@marek-safar @vitek-karas - thoughts?

Reply:
No, I don't see any benefit of that work. Why?

Reply:
It would be for the same reason why we have moved e.g. ASP.NET from the aspnet org to dotnet org. The primary reasons are consistency and public perception.

Other more technical reasons include permission management or complications moving issues between repos when they are in different orgs.